components/HomePageIdeaPicture.tsx

```diff
@@ -0,0 +1,9 @@
+const HomePageIdeaPicture = () => {
+    return (
+        <div>
+            <img src="/home_page_picture.png" className="h-auto" alt="footer-logo" />
```

Comment:
We already have imported icon for light on left side of component [here](https://github.com/lookagit/appvironments/blob/639701a5eb82a1ab963e0c2a6c2a0cde21387eda/public/images/appvironments_application_idea.png)
instead of importing new picture `/home_page_picture.png`.